src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -75,6 +77,24 @@ Qgs3DMapCanvasWidget::Qgs3DMapCanvasWidget( const QString &name, bool isDocked )
 
   toolBar->addAction( QgsApplication::getThemeIcon( QStringLiteral( "mActionZoomFullExtent.svg" ) ), tr( "Zoom Full" ), this, &Qgs3DMapCanvasWidget::resetView );
 
+  // Editing toolbar
+  mEditingToolBar = new QToolBar( this );
+  mEditingToolBar->setVisible( false );
+
+  QAction *actionPointCloudChangeAttributeTool = mEditingToolBar->addAction( QIcon( QgsApplication::iconPath( "mActionSelectPolygon.svg" ) ), tr( "Change Point Cloud Attribute" ), this, &Qgs3DMapCanvasWidget::changePointCloudAttribute );
+  actionPointCloudChangeAttributeTool->setCheckable( true );
+
+  mEditingToolBar->addWidget( new QLabel( tr( "Attribute:" ) ) );
```

Comment:
I would suggest to avoid colons in the labels (I believe that's the case elsewhere in QGIS GUI)